docs/benchmarks.md

```diff
@@ -105,20 +105,24 @@ The benchmark can be executed with the following syntax to execute the query and
 results to the driver.
 
 ```scala
-TpcdsLikeBench.collect(spark, "q5", iterations=3)
+import com.nvidia.spark.rapids.tests.common._
+val benchmark = new BenchmarkRunner(TpcdsLikeBench)
+benchmark.collect(spark, "q5", iterations=3)
 ```
 
 The benchmark can be executed with the following syntax to execute the query and write the results 
 to Parquet. There are also `writeCsv` and `writeOrc` methods for writing the output to CSV or ORC 
 files.
 
 ```scala
-TpcdsLikeBench.writeParquet(spark, "q5", "/data/output/tpcds/q5", iterations=3)
+import com.nvidia.spark.rapids.tests.common._
```

Comment:
Curious if the `common` package is worth it.  Since this is going to be visible to users, less typing is a plus.  Wondering if this should be simplified to `import com.nvidia.spark.rapids.tests.BenchmarkRunner`.  We can still place internal classes in a `common` package if that's needed at some point.